troubleshooting/ETW/Readme.md

```diff
@@ -127,43 +127,69 @@ This file will not exceed the configured max size and will be circularly overwri
 
 #### Configuration
 
-Configuration is controlled by a file named `ApplicationInsightsDiagnostics.json`.
+Configuration is controlled by a file named `ApplicationInsightsDiagnostics.json` or environment variables.
 The configuration file must be no more than 4 KiB, otherwise only the first 4 KiB of content will be read.
 
-**To enable self-diagnostics**, go to the [current working directory](https://en.wikipedia.org/wiki/Working_directory) of your process and create a configuration file.
+#### Two ways to enable self-diagnostics
```

Comment:
Should this be at the same level as the `#### Configuration` above?

Reply:
I've added an extra `#` to lower the level of the title.